lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2932,7 +2932,7 @@ ParseResult FIRStmtParser::parseWire() {
         getAnnotations(getModuleTarget() + ">" + id, startTok.getLoc(),
                        moduleContext.targetsInModule, type);
 
-  auto result = builder.create<WireOp>(type, id, annotations);
+  auto result = builder.create<WireOp>(type, id, annotations, StringAttr{});
```

Comment:
If the wire has name, do we need the symbol ?

Reply:
I think these things are separate. Right now there is now way for the parser to attach a symbol to a wire (or reg, node, etc.). It will just populate the name and not assign a symbol. The symbols are then added in transformation passes as they need to keep a reference to certain ops.

Reply:
Names != symbols.  (unlike a classic C compiler).  We are using symbols only for reference, never for naming (except modules, for now).